class/class4.1/src/Pizza.cpp

```diff
@@ -0,0 +1,19 @@
+#include "Pizza.hpp"
+
+void Pizza::addIngredient(Ingredient&& ingredient) {
+    this->ingredients.push_back(std::move(ingredient));
+}
+
+std::vector<Ingredient> Pizza::getIngredients() const {
```

Comment:
nodiscard seems to be reasonable for this case as if someone asks for a big object like vector then it is very strange if that vector will be ignored